### PR TITLE
complete the testcase to valicate the pagination

### PR DESCRIPTION
**Pull request by NhuyNhuy63**

### CODE REVIEW

File: src/02-ttts2023-agu-pagination-nhi-le.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+*          3.2.4. push the code to the new branch
+*          3.2.5. create a pull request in github to merge the new branch to "develop"
+*/
+import { Selector } from "testcafe";
+
+fixture("ttts2023-agu").page("https://www.agu.edu.vn/vi/tim-kiem");
+
+test("search for 'tuyển sinh' and validate some results appear", async (t) => {
+    const searchInputSelector = Selector(".search .input-group .search");
+    const pagination = Selector(".main .page-item .page-link").nth(2);
```
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu-pagination-nhi-le.test.js]: page2Button

File: src/02-ttts2023-agu-pagination-nhi-le.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    const firsttext = Selector(".main .container .blog-details >h5 >a");
```
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu-pagination-nhi-le.test.js]: revise this selector

Comment (liamnguyeniaa) [inline src/02-ttts2023-agu-pagination-nhi-le.test.js]: firstPostTitle

File: src/02-ttts2023-agu-pagination-nhi-le.test.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    const FirstText = await firsttext.textContent;
```
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu-pagination-nhi-le.test.js]: firstPostTitleContent

File: src/02-ttts2023-agu-pagination-nhi-le.test.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    await t.typeText(searchInputSelector, "tuyển sinh").pressKey("enter");
+  
+    
+    await t
+      .expect(Selector(".main .thong-tin .blog-details .btn").exists)
+      .ok("At least one search result should exist");
+    
+    await t.expect(pagination.exists).ok("pagination should exist");
+    await t.click(pagination);
+    const secondtext = Selector(".main .container .blog-details >h5 >a");
```
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu-pagination-nhi-le.test.js]: firstPostTitleOnPage2

Comment (liamnguyeniaa) [reply src/02-ttts2023-agu-pagination-nhi-le.test.js]: revise the selector

**Review by liamnguyeniaa — CHANGES_REQUESTED** [review]
comments

File: src/02-ttts2023-agu-pagination-nhi-le.test.js [hunk 5]
```diff
@@ -26,8 +27,8 @@ test("search for 'tuyển sinh' and validate some results appear", async (t) =>
...
     
     await t.expect(pagination.exists).ok("pagination should exist");
-    await t.click(pagination);
-    const secondtext = Selector(".main .container .blog-details >h5 >a");
-    const SecondText = await secondtext.textContent;
-    await t.expect(FirstText).notEql(SecondText);
+    await t.click(page2Button);
+    const firstPostTitleOnPage2 = Selector(".main .container .blog-details:first-child");
+    const firstPostTitleOnPage2Content = await firstPostTitleOnPage2.textContent;
+    await t.expect(firstPostTitleContent).notEql(firstPostTitleOnPage2Content);
```
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu-pagination-nhi-le.test.js:38]: Need to test the next and prev button as well.

**Review by liamnguyeniaa — CHANGES_REQUESTED** [review]
only minor comments left

**Review by tvteo — APPROVED** [review]

File: src/02-ttts2023-agu-pagination-nhi-le.test.js (hunk 1 above)
Comment (NhuyNhuy63) [reply src/02-ttts2023-agu-pagination-nhi-le.test.js]: done

File: src/02-ttts2023-agu-pagination-nhi-le.test.js (hunk 2 above)
Comment (NhuyNhuy63) [reply src/02-ttts2023-agu-pagination-nhi-le.test.js]: done

File: src/02-ttts2023-agu-pagination-nhi-le.test.js (hunk 3 above)
Comment (NhuyNhuy63) [reply src/02-ttts2023-agu-pagination-nhi-le.test.js]: done

File: src/02-ttts2023-agu-pagination-nhi-le.test.js (hunk 4 above)
Comment (NhuyNhuy63) [reply src/02-ttts2023-agu-pagination-nhi-le.test.js]: done

File: src/02-ttts2023-agu-pagination-nhi-le.test.js (hunk 2 above)
Comment (NhuyNhuy63) [reply src/02-ttts2023-agu-pagination-nhi-le.test.js]: done

**Review by liamnguyeniaa — APPROVED** [review]

**Pull request merged**